src/playground/sample_data_tooling/sample_data_tests/sample_data_ingestor_tests/unit-test/sample_data_ingestor_unit_test.py

```diff
@@ -0,0 +1,132 @@
+"""
+Copyright OpenSearch Contributors
+SPDX-License-Identifier: Apache-2.0
+"""
+
+from opensearchpy import OpenSearch
+
+# Standard libraries
+from json import loads, load
+import pytest
+import sys
+import os
+
+# Adds the directory "/sample_data_tooling" to sys.path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))))
+from sample_data_ingestor.sample_data_ingestor import ingest, build_request_body
+
+# Constants
+INDEX_NAME = "ingest-test"
+DIR_PATH = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Can we make this simple for the path? thanks

Reply:
Done, made `sys.path` more readable.